USFMConverter/UI/Pages/FileView.axaml.cs

```diff
@@ -86,6 +89,23 @@ private async void OnBrowseClick(object? sender, RoutedEventArgs e)
             }
         }
 
+        private void OnSelectAllClick(object? sender, RoutedEventArgs e)
+        {
+            if (selected)
```

Comment:
I don't nessicarily like the amount of duplication between this and the other function but it isn't that bad. Nothing I'm going to block on.